### ISSUE TITLE
Config option to disable explosion of Tiberium

### ISSUE THREAD

**Issue by TheSpiced** [TeamFRM/TAIGA#115]
Please let us turn this off. My first reaction was to quit and uninstall this mod, figured I'd try this approach first.

I love the added progression, but it's so hostile as soon as you're able to access any part of it. There's also no reference to this happening anywhere in the mod description that I can find?

This is just my 2 cents, any discussion about this is is more than welcome.

**Comment by cyruscm** [comment]
Agreed. It caused a cascading effect that shut down my server by triggering runaway phases

`[23:20:59] [Server thread/ERROR] [Sponge/]: /*******************************************************************************************************************/
[23:20:59] [Server thread/ERROR] [Sponge/]: /*******************************************************************************************************************/
[23:20:59] [Server thread/ERROR] [Sponge/]: /*                                                 Switching Phase
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*******************************************************************************************************************/
[23:20:59] [Server thread/ERROR] [Sponge/]: /* Detecting a runaway phase! Potentially a problem
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /* where something isn't completing a phase!!!
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*   Entering Phase : GeneralPhase{}
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*   Entering State : org.spongepowered.common.event.tracking.phase.general.ExplosionState@59bc669f                */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*         - Name: CapturedExplosion
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*           Object: CaptureExplosion{explosion=null}
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*         - Name: CapturedEntities
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*           Object: CapturedEntitiesSupplier{Captured=0}
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*         - Name: CapturedBlocks
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*           Object: CapturedBlocksSupplier{Captured=0}
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*         - Name: Source
                          */
[23:20:59] [Server thread/ERROR] [Sponge/]: /*           Object: WorldServer{LevelName=DIM-1, DimensionId=-1, DimensionType=minecraft:nether}                  */
[23:20:59] [Server thread/ERROR] [Sponge/]: /* Phases remaining :`
